docs/spec.md

```diff
@@ -101,14 +102,34 @@ Each JSON message must be an object containing a field called `op` which identif
 - `op`: string `"status"`
 - `level`: 0 (info), 1 (warning), 2 (error)
 - `message`: string
+- `id`: string | undefined. Optional identifier for the status message. Newer status messages with the same identifier should replace previous messages. [clearStatus](#clear-status) can reference the identifier to indicate a status message is no longer valid.
```

Comment:
seems odd to me that the ID is optional here but not optional for `clearStatus`

Reply:
The `id` field has to be optional here to keep the `status` operation backwards compatible.